docker/scripts/set-sequencer-url.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-console */
+
+const createRollupOutput = require('../deploymentOutput/create_rollup_output.json');
```

Comment:
can we add a TODO doc block here that we should:

1. make these a input param
2. convert this script to a hardhat task
3. create a npm command that will preconfig these params

same for other scrtips e.g. set up committe

Reply:
OK